src/components/UserWarnings/LoggedOut/LoggedOut.tsx

```diff
@@ -1,16 +1,26 @@
-import React from 'react'
+import React, { ReactElement, useEffect, useState } from 'react'
 import { useSelector } from 'react-redux'
 import { BodyLong, Button, Heading, Modal } from '@navikt/ds-react'
 
 import { RootState } from '../../../state/store'
 
-function LoggedOut(): JSX.Element | null {
+function LoggedOut(): ReactElement {
     const loggedOut = useSelector((state: RootState) => state.metadata.loggedOut)
+    const [isModalOpen, setIsModalOpen] = useState(false)
 
-    if (!loggedOut) return null
+    useEffect(() => {
+        if (loggedOut && !isModalOpen) {
+            setIsModalOpen(true)
+        }
+    }, [loggedOut, isModalOpen, setIsModalOpen])
```

Comment:
man burde unngå å bruke effects for å "inferre" en verdi. Det er bedre å bare bruke verdien direkte i modal, siden det bare er en boolean

```
 <Modal
  open={loggedOut}
```

da trenger man ikke useState og useEffect :smile: